platform-emf/src/main/resources/com/softicar/platform/emf/emf-form-style.css

```diff
@@ -1,8 +1,12 @@
 
 .EmfForm {
+	width: 100%; 
```

Comment:
I am paranoid, what does that `100%` accomplish?

Reply:
it ensures that the form in the popup will always take up the whole width of the popup
There was a style request in the past where popups with EXTREMELY long titles looked weird because the common actions etc werent on the right side

Reply:
![image](https://user-images.githubusercontent.com/90852097/159660866-ed83dc81-e774-435d-834b-17323894b784.png)
![image](https://user-images.githubusercontent.com/90852097/159660895-5321c446-50c6-4cbf-938d-8b123a738d3e.png)